contracts/DependencyAudit.cdc

```diff
@@ -19,6 +19,8 @@ access(all) contract DependencyAudit {
 
     access(all) event PanicOnUnstagedDependenciesChanged(shouldPanic: Bool)
 
+    access(all) event BlockBoundariesChanged(boundaries: Boundaries?)
```

Comment:
I think emitting the boundary values `start` and `end` as optionals would result in more human readable and queryable events

Reply:
yes but its more difficult to capture the case where the boundaries are removed.

Reply:
What do you think about emitting `start` and `end` as `nil` in that case?

Reply:
changed: 32e6f4e